tests/test_qaneeded.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbug.models.qaneeded import QANeededModel
+
+
+def test_get_qaneeded_labels():
+    model = QANeededModel()
+    classes = model.get_labels()
+    assert not classes[1389220]
+    assert classes[1389223]
```

Comment:
Can you also add a test with qawanted?

Reply:
Done.

Reply:
And we should test the case where qe-verify is in the flags.

Also, please add a rollback test.